airbyte-integrations/connectors/source-s3/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: 69589781-7828-43c5-9f63-8925b1c1ccc2
-  dockerImageTag: 3.1.8
+  dockerImageTag: 4.0.1
```

Comment:
why `4.0.1` instead of `4.0.0`?

Reply:
Because of a bug where I could not release a dev version on 4.0.0 (see https://airbytehq-team.slack.com/archives/C03VDJ4FMJB/p1692394696511219?thread_ts=1692394086.692969&cid=C03VDJ4FMJB)